src/System/Wlog/Wrapper.hs

```diff
@@ -61,11 +62,17 @@ streamHandlerWithLock lock h sev = do
 -- 3. Applies `setSeverity` to given loggers. It can be done later using
 -- `setSeverity` directly.
 initTerminalLogging :: MonadIO m
-                    => Bool  -- ^ Show time?
+                    => (UTCTime -> Text)
+                    -> Bool  -- ^ Show time?
                     -> Bool  -- ^ Show ThreadId?
                     -> Maybe Severity
                     -> m ()
-initTerminalLogging isShowTime isShowTid (fromMaybe Warning -> defaultSeverity) = liftIO $ do
+initTerminalLogging
+    timeF
```

Comment:
I don't get it. Why does it only apply to terminal logging? I expected `timeF` to be the field of logging configuration or something... Can you elaborate?

Reply:
@volhovM I don't understand how you can specify arbitrary `UTCTime -> Text` function inside logging configuration. I tried with custom templates like `%Y-%m-%d %H:%M:%S%Q %Z` which you actually can specify in configuration — you didn't like it because you want `UTCTime -> Text` function. But if you want arbitrary function you can't write it inside logger configuration like you could with this pattern.

> Why does it only apply to terminal logging?

It doesn't apply only to terminal loggin. Files affected as well. See here:

https://github.com/serokell/log-warper/blob/3c7df72add4eb87a0ee1cdad40b07d1a7d4348b5/src/System/Wlog/Launcher.hs#L100

Reply:
>I don't understand how you can specify arbitrary UTCTime -> Text function inside logging configuration. 

```
{ 
...
, timeFormatter :: UTCTime -> Text
... }
```
I don't really care if we can `Show` a configuraiton but it might be wrong approach. Anyway, ok.